### PR TITLE
fix a broadcast exploit in AnimationPacket

### PR DESCRIPTION
**Pull request by liulihaocai**
A "hacked client" can send AnimationPacket with illegal Action, like CRITICAL_HIT.
If the illegal client send too many these packets(like 1000 pps?), viewers' client will drop fps or crash

### CODE REVIEW

**Comment by Superice666** [comment]
惊！琉璃大佬重出江湖！

**Comment by smartcmd** [comment]
惊！琉璃大佬重出江湖！

File: src/main/java/cn/nukkit/Player.java [hunk 1]
```diff
@@ -3169,13 +3169,21 @@ public void onCompletion(Server server) {
 
                     AnimatePacket animatePacket = (AnimatePacket) packet;
                     PlayerAnimationEvent animationEvent = new PlayerAnimationEvent(this, animatePacket);
```
Comment (Superice666) [inline src/main/java/cn/nukkit/Player.java:3171]: 我觉得可以把事件创建放到break后面去

**Review by Superice666 — COMMENTED** [review]
没啥大问题，事件的逻辑或许可以考虑改下

File: src/main/java/cn/nukkit/Player.java (hunk 1 above)
Comment (liulihaocai) [reply src/main/java/cn/nukkit/Player.java:3171]: 确实

**Review by lt-name — APPROVED** [review]

**Review by Superice666 — APPROVED** [review]

**Pull request merged**